service/src/integrationTest/kotlin/fi/espoo/evaka/linkity/MockLinkityClient.kt

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2017-2025 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.linkity
+
+import fi.espoo.evaka.shared.domain.FiniteDateRange
+
+class MockLinkityClient : LinkityClient {
+    private val shifts: List<Shift>
+
+    constructor(shifts: List<Shift>) {
+        this.shifts = shifts
+    }
```

Comment:
Voi myös sanoa vaan:

```suggestion
class MockLinkityClient(private val shifts: List<Shift>) : LinkityClient {
```